backend/src/crd/kubernetes/v2beta1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2025.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v2beta1 contains API Schema definitions for the  v2beta1 API group.
+// +kubebuilder:object:generate=true
+// +groupName=kubeflow.org
```

Comment:
The group name should be `pipelines.kubeflow.org` to match what is in the design, match the existing SubjectAccessReview checks, and to prevent collisions with other Kubeflow CRDs.